buildbot/master/files/config/master.cfg

```diff
@@ -170,7 +170,7 @@ c['builders'] = [
     DynamicServoBuilder("arm64", CROSS_SLAVES, envs.build_arm64),
     DynamicServoBuilder("linux-dev", LINUX_SLAVES, envs.build_linux),
     DynamicServoBuilder("linux-nightly", LINUX_SLAVES, envs.build_linux),
-    DynamicServoBuilder("linux-rel", LINUX_SLAVES, envs.build_linux),
+    DynamicServoBuilder("linux-rel", LINUX_SLAVES, envs.build_linux_rel_debug_assert),
```

Comment:
You'll need to move the last argument to the next line to fit inside the line length limit.